runners/google-cloud-dataflow-java/pom.xml

```diff
@@ -41,21 +38,35 @@
     <beam.version>0.1.0-incubating-SNAPSHOT</beam.version>
     <timestamp>${maven.build.timestamp}</timestamp>
     <maven.build.timestamp.format>yyyy-MM-dd HH:mm</maven.build.timestamp.format>
-    <dataflow>com.google.cloud.dataflow</dataflow>
-    <runIntegrationTestOnService>false</runIntegrationTestOnService>
-    <testParallelValue>none</testParallelValue>
-    <testGroups></testGroups>
-    <dataflowProjectName></dataflowProjectName>
   </properties>
 
+  <!-- Disable integration tests unless told configured by a test flag.
```

Comment:
This profile disables automatic execution of RunnableOnService integration tests on Google Cloud Dataflow service. However, the tests will be executed if `runnableOnServicePipelineOptions` is specified.